### PR TITLE
updated to redash 10.0, redis 7, postgres 14

### PR DESCRIPTION
**Pull request by thomas-wroblewski**
Updated to Redash 10.

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -11,15 +10,17 @@ services:
     <<: *redash-service
     command: server
     ports:
-      - "5000:5000"
+      - "5001:5000"
```
Comment (kdabir) [inline docker-compose.yml]: why do we need to change the port to 5001?

**Comment by kdabir** [comment]
thanks for the PR, I have added a comment. once resolved I am happy to merge it.

File: docker-compose.yml (hunk 1 above)
Comment (thomas-wroblewski) [reply docker-compose.yml]: I had something else running locally on port 5000 - ive adjusted the PR accordingly back to port 5000

File: docker-compose.yml [hunk 2]
```diff
@@ -14,12 +13,14 @@ services:
...
       REDASH_WEB_WORKERS: 4
+    links:
+      - postgres:postgres
   scheduler:
     <<: *redash-service
     command: scheduler
-    environment:
-      QUEUES: "celery"
-      WORKERS_COUNT: 1
+    # environment:
```
Comment (kdabir) [inline docker-compose.yml:21]: doesn't the updated scheduler service need queue name and worker count?

Comment (thomas-wroblewski) [reply docker-compose.yml:21]: No. That’s a change in v10. I followed the upgrade path documented here

https://github.com/getredash/redash/releases/tag/v10.0.0

**Comment by kdabir** [comment]
thanks for the contribution 👍

**Pull request merged**